pkg/cmdutil/args.go

```diff
@@ -21,6 +21,24 @@ func MinimumArgs(n int, msg string) cobra.PositionalArgs {
 	}
 }
 
+func ExactArgs(n int, msg string) cobra.PositionalArgs {
+	if msg == "" {
+		return cobra.MinimumNArgs(1)
+	}
```

Comment:
I think you can get rid of this fallback. We'll just always make sure to pass a non-blank string.